homeassistant/components/velbus/climate.py

```diff
@@ -66,6 +64,14 @@ def current_temperature(self) -> int | None:
         """Return the current temperature."""
         return self._channel.get_state()
 
+    @property
+    def hvac_mode(self) -> HVACMode:
+        """Return the current hvac mode based on cool_mode message."""
+        return {
+            True: HVACMode.COOL,
+            False: HVACMode.HEAT,
+        }.get(self._channel.get_cool_mode())
```

Comment:
Please move the map to a module level constant:

```suggestion
        # Map velbus cool mode to Home Assistant HVACMode
        COOL_MODE_MAP = {
            True: HVACMode.COOL,
            False: HVACMode.HEAT,
        }
        
        ...

        return COOL_MODE_MAP.get(self._channel.get_cool_mode())
```